pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -250,16 +250,16 @@ func (p *PriorityQueue) Add(pod *v1.Pod) error {
 	defer p.lock.Unlock()
 	pInfo := p.newQueuedPodInfo(pod)
 	if err := p.activeQ.Add(pInfo); err != nil {
-		klog.Errorf("Error adding pod %v to the scheduling queue: %v", nsNameForPod(pod), err)
```

Comment:
can you check if this `nsNameForPod` function is used anywhere else? From my search it was only used by scheduling_queue and scheduling_queue_test, so it can be removed

Reply:
Good advice, I have modified it.